package-lock.json

```diff
@@ -1,9949 +1,8 @@
 {
   "name": "depcheck",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Looks like you are using some outdated npm version

Reply:
I just fixed that. I was using the one that ships w/ Node LTS (i.e. 14) – that is npm v6.